doc/workshops/financial.rst

```diff
@@ -2,4 +2,1065 @@
 Financial Data Analysis with Spyder
 ###################################
 
-This is a work in progress. The workshop for Financial Data Analysis with Spyder will be available soon!
+By the end of this workshop participants will be able to use Spyder effectively for  applying some normative financial theories and models to assemble a portfolio of assets in a way that maximizes the expected return for a given level of risk. In this way, statistically-based tools are used to construct investment portfolios.
+
+
+
+=============
+Prerequisites
+=============
+
+To follow this workshop we recommend that you have a basic knowledge of Python. You can visit `The Python Tutorial`_ to learn the basics of this programming language or to refresh your knowledge of Python.
+
+.. _The Python Tutorial: https://docs.python.org/3/tutorial/
+
+You will also need to have Spyder installed. More information about installation in :doc:`installation guide<../installation>`.
```

Comment:
I think they need Anaconda since they need them to create environments and install packages